### PR TITLE
update bodhi multisigner

### PR DESCRIPTION
**Pull request by ermalkaleci**

### CODE REVIEW

File: bodhi/src/MultiSigner.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import type { Signer, SignerResult } from '@polkadot/api/types';
+import type { SignerPayloadJSON } from '@polkadot/types/types';
+
+export class MultiSigner implements Signer {
+  #singers: Record<string, Signer>;
```
Comment (ermalkaleci) [inline bodhi/src/MultiSigner.ts:5]: Typo

**Review by shunjizhan — APPROVED** [review]
it would be nice to have some examples/tests for multiSigner later

**Comment by ermalkaleci** [comment]
> it would be nice to have some examples/tests for multiSigner later

All the tests are using utils/getTestUtils so it's basically tested

**Comment by shunjizhan** [comment]
wallets from util is `SubstrateWallet` instead of `multiSigner`, right?

**Comment by shunjizhan** [comment]
but multisigner is not actually used anywhere so should be fine

**Comment by ermalkaleci** [comment]
This will fix the issue with creating multiple BodhiSigner. Instead of replacing signer it will append to multisigner. So now we can create multiple wallets with different signer. Doesn't need to be SubstrateSigner

**Pull request merged**